fix(*): "We"/"our"/etc. to "you"/"your"/etc. (#661)
discordjs/guide: guide/additional-info/async-await.md

### Execution with discord.js code

> After knowing how Promises work and what they are for, as well as the theory, let's look at an example in which handles multiple Promises. Let's say you want to react with letters (regional indicators) in a specific order. For this example, here's a basic template for a discord.js bot with some ES6 adjustments.

Reviewer: ```suggestion
Now that you know how Promises work and what they are used for, let's look at an example that handles multiple Promises. Let's say you want to react with letters (regional indicators) in a specific order. For this example, here's a basic template for a discord.js bot with some ES6 adjustments.
```
